larray/core/axis.py

```diff
@@ -204,10 +204,6 @@ def by(self, length, step=None, name_template=None):
             length of groups
         step : int, optional
             step between groups. Defaults to length.
-        name_template : str, optional
```

Comment:
hmmmm, now that I've seen some of the code in the reports for demo, I guess this could be useful after all. The question is whether they actually need that particular notation, or if they could do with the default one?